man/calcGDPpppPast.Rd

```diff
@@ -15,8 +15,8 @@ GDP PPP(ICP11) in million USD05 equivalents
 \description{
 \ifelse{html}{\href{https://lifecycle.r-lib.org/articles/stages.html#deprecated}{\figure{lifecycle-deprecated.svg}{options: alt='[Deprecated]'}}}{\strong{[Deprecated]}}
 
-This function is defunct. Use \code{\link[mrdrivers:calcGDP]{mrdrivers::calcGDPPast()}} instead.
```

Comment:
I thought that we do not have calcGDPpast any more and should use calcGDP. Why did you change that?

Reply:
You're right. I did not changes that intentionally but had outdated packages. Changes to most `.Rd` files vanished after updates.